src/music_module/spec/music_album_spec.rb

```diff
@@ -0,0 +1,17 @@
+require_relative '../music_album'
+
+describe MusicAlbum do
+  it('should test for on_spotify instance varaible') do
+    music_album = MusicAlbum.new('10-01-2023', true)
+    expect(music_album.on_spotify).to eql(true)
+  end
+
+  it('should test for publish date instance varaible') do
+    music_album = MusicAlbum.new('10-01-2023', true)
+    expect(music_album.publish_date).to eql('10-01-2023')
+  end
+  it('should test for archived instance varaible') do
+    music_album = MusicAlbum.new('10-01-2023', true)
+    expect(music_album.archived).to eql(false)
+  end
+end
```

Comment:
This class has another private method called `can_be_archived`,
https://github.com/Fayob/catalog_of_my_things/blob/49108f2063c9abc4b095d872a634300058cabb66/src/music_module/music_album.rb#L13-L19

you can test it by triggering the public method `move_to_archive` on the parent class; `item`
https://github.com/Fayob/catalog_of_my_things/blob/49108f2063c9abc4b095d872a634300058cabb66/src/item.rb#L36-L38